src/types.ts

```diff
@@ -85,6 +85,20 @@ export interface VitePWAOptions {
    * @default "base" options from Vite
    */
   base?: string
+  /**
+   * `public` resources to be added to the PWA manifest.
+   *
+   * You don't need to add `manifest` icons here, it will be auto included.
+   *
+   * The `public` directory will be resolved from Vite's `publicDir` option directory.
+   */
+  include: string | string[] | undefined
```

Comment:
```suggestion
  includeAssets: string | string[] | undefined
```

Maybe this way could be more explicit, WDYT?

Reply:
yeah, I forgot to mention renaming options...

Reply:
rename done